test/pool-utils/PoolTestFixture.ts

```
@@ -176,8 +176,8 @@ export abstract class PoolTestFixture {
   /**
    * Deposit BackingTokens into TempusPool
    */
-  async depositBT(user:Signer, backingTokenAmount:NumberOrString, recipient:SignerOrAddress = user): Promise<Transaction> {
-    return this.tempus.controller.depositBacking(user, this.tempus, backingTokenAmount, recipient);
+  async depositBT(user:Signer, backingTokenAmount:NumberOrString, recipient:SignerOrAddress = user, ethValue: NumberOrString = 0): Promise<Transaction> {
```

Since we do an override in `LidoTestPool.ts`, what will happen to that? The last argument is silently ignored?

---

Yes. Though this test is not affected because of the `backingToken != 0` condition.

---

Do you have any strong opinion for not just going with the first commit and then arguing about the second later?